pkg/crc/preflight/preflight_linux.go

```diff
@@ -97,13 +97,17 @@ var libvirtNetworkPreflightChecks = [...]Check{
 }
 
 var vsockPreflightChecks = Check{
-	configKeySuffix:  "check-vsock",
-	checkDescription: "Checking if vsock is correctly configured",
-	check:            checkVsock,
-	fixDescription:   "Checking if vsock is correctly configured",
-	fix:              fixVsock,
+	configKeySuffix:    "check-vsock",
+	checkDescription:   "Checking if vsock is correctly configured",
+	check:              checkVsock,
+	fixDescription:     "Checking if vsock is correctly configured",
+	fix:                fixVsock,
+	cleanupDescription: "Removing vsock configuration",
```

Comment:
In this commit you are also adding the cleanup logic, shouldn't that part of different commit message?

Reply:
created new commit for the cleanup part, thanks!